Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Requirements for container registry
+type: Decision Record
+status: Draft
+track: KaaS
+---
+
+# Introduction  
+
+A container registry is a repository or collection of repositories used to store and 
+access container images. Container registries are used by the teams to store the 
+results of build processes and to maintain freely available images. They're acting as 
+the intermediary for sharing container images between systems. They could be connected 
+directly to the container orchestration platforms like Kubernetes.
+
+The container registries could be publicly accessible e.g. Docker Hub or could be
+self-hosted or hosted by the cloud service providers (CSP). These container registries
+may apply various access control mechanisms to restrict public access and make them
+private. Both solutions offer a wide range of features that may or may not attract
+potential users and CSPs.
+
+# Motivation
+
+Our motivation is that there are use cases, where CSP would like to be able to offer
+customers to store and access their container images using the SCS-compliant private
+container registry. The specific use cases should be discussed, but overall the CSPs
+could offer a private container registry as a service or the CSPs could offer a recipe
+for customers to deploy the private registry themselves utilizing the CSP infrastructure.
+In both cases, the private container registry should be SCS-compliant and should fulfill
+a set of needed requirements e.g. for security and privacy.
+
+The idea here and the purpose of this document is to specify what requirements a 
```

Comment:
`The idea here and the purpose of this document` could be replaced with `The idea and purpose of this document`.